tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -15,18 +15,18 @@
     <string name="btnITPDStart" translatable="false">Start I2P</string>
     <string name="btnITPDStop" translatable="false">Stop I2P</string>
     <string name="tvITPDStop">I2P Stopped</string>
-    <string name="tvITPDRunning">I2P Running</string>
-    <string name="tvITPDStarting">I2P Starting</string>
-    <string name="tvITPDStopping">I2P Stopping</string>
-    <string name="tvITPDInstalling">I2P Installing</string>
+    <string name="tvITPDRunning">I2P Running…</string>
+    <string name="tvITPDStarting">I2P Starting…</string>
+    <string name="tvITPDStopping">I2P Stopping…</string>
+    <string name="tvITPDInstalling">I2P Installing…</string>
```

Comment:
```suggestion
    <string name="tvITPDInstalling">Installing I2P…</string>
```
I agree this is iffy. Ideally it should be this.
The difference is "~Installation of I2P" vs "~Now installing I2P" in translation.
Does this lead to a focus shift when possibly "I2P" changes from one end of the sentence to the other?